.github/workflows/merge.yml

```diff
@@ -44,12 +44,13 @@ jobs:
       - name: Version Beta
         if: github.ref == 'refs/heads/develop'
         run: |
-          npm run build
+          node ./scripts/update-package-json-workspaces.js
```

Comment:
Ok so heres a key feature. At the moment changesets and nx aren't 100% compatible because nx has the convention of outputting into `${workspaceRoot}/dist/packages/[my-package]/*`

and changesets doesnt understand this at the moment (because they use something called `@manypkg` which doesnt know this information out of the box. theres ongoing conversations about this but my understanding is that changesets will essentially look at the workspace to find the package.

so if we modify the workspaces key in the package.json to have `dist/` in front it should identify the proper package location.